rpcserver.go

```diff
@@ -1044,19 +1060,29 @@ func (b blockDisconnected) notificationCmds(w *Wallet) []btcjson.Cmd {
 
 func (c txCredit) notificationCmds(w *Wallet) []btcjson.Cmd {
 	blk := w.Manager.SyncedTo()
-	ltr, err := txstore.Credit(c).ToJSON("", blk.Height, activeNet.Params)
+	acctName := waddrmgr.DefaultAccountName
+	if creditAccount, err := w.CreditAccount(txstore.Credit(c)); err == nil {
+		// acctName is defaulted to DefaultAccountName in case of an error
+		acctName, _ = w.Manager.AccountName(creditAccount)
+	}
+	ltr, err := txstore.Credit(c).ToJSON(acctName, blk.Height, activeNet.Params)
 	if err != nil {
 		log.Errorf("Cannot create notification for transaction "+
 			"credit: %v", err)
 		return nil
 	}
-	n := btcws.NewTxNtfn("", &ltr)
+	n := btcws.NewTxNtfn(acctName, &ltr)
 	return []btcjson.Cmd{n}
 }
 
 func (d txDebit) notificationCmds(w *Wallet) []btcjson.Cmd {
 	blk := w.Manager.SyncedTo()
-	ltrs, err := txstore.Debits(d).ToJSON("", blk.Height, activeNet.Params)
+	acctName := waddrmgr.DefaultAccountName
+	if debitAccount, err := w.DebitAccount(txstore.Debits(d)); err == nil {
```

Comment:
I would also just use "" here (this is an old api and will eventually be replaced) instead of looking up the account.  This is the last call to DebitAccount, so with this gone, you can remove that func as well.